src/libs/actions/Report.ts

```diff
@@ -2168,6 +2168,13 @@ function deleteReport(reportID: string) {
 
     Onyx.multiSet(onyxData);
 
+    Object.values(reportActionsForReport ?? {}).forEach((reportAction) => {
+        if (!reportAction.childReportID) {
+            return;
+        }
+        deleteReport(reportAction.childReportID);
```

Comment:
We should probably not always delete child reports. Can we put this behind a `shouldDeleteChildReports` flag?

Reply:
Done